Sources/PointFree/MinimalNavView.swift

```diff
@@ -101,6 +102,8 @@ private let logInLinkView = View<(NavStyle.MinimalStyle, Route?)> { style, curre
 
 private func gitHubLinkType(for style: NavStyle.MinimalStyle) -> GitHubLinkType {
   switch style {
+  case .black:
+    return .white
```

Comment:
I added a new style of minimal nav for a completely black nav. this will need some separate work because @amotion's design calls for a completely different nav, but I didn't want to tackle that yet.